azure-pipelines.yml

```diff
@@ -0,0 +1,102 @@
+# ASP.NET
+# Build and test ASP.NET projects.
+# Add steps that publish symbols, save build artifacts, deploy, and more:
+# https://docs.microsoft.com/azure/devops/pipelines/apps/aspnet/build-aspnet-4
+
+trigger:
+  batch: true
+  branches:
+    include:
+    - '*'
+  tags:
+    include:
+    - 'v*'
+    - '*-deploy'
+
+pr:
+- 'development'
+- 'master'
+
+pool:
+  vmImage: 'windows-latest'
+
+steps:
+# -------------------
+# Install Dependencies
+# -------------------
+- task: UseDotNet@2
+  displayName: Require .NET 2.1
+  inputs:
+    packageType: 'runtime'
+    version: '2.1.x'
+
+- task: UseDotNet@2
+  displayName: Require .NET 3.1 SDK for Build Process
+  inputs:
+    packageType: 'sdk'
+    version: '3.1.x'
+
+- task: NuGetCommand@2
+  displayName: 'Authenticate with NuGet feed'
+  inputs:
+   command: custom
+   feedsToUse: config
+   arguments: sources update -Name "$(bulkwriterfeed)" -Username "bulkWriter" -Password "$(bulkwriterfeedpassword)" -StorePasswordInClearText -ConfigFile nuget.config
+
+- task: PowerShell@2
+  inputs:
+    targetType: inline
+    script: $(Build.SourcesDirectory)/setup.ps1
```

Comment:
Just a minor point, you shouldn't need `$(Build.SourcesDirectory)` here, everything will by default be relative to the checkout folder.